client/src/components/list/UserList.tsx

```diff
@@ -0,0 +1,27 @@
+import { Avatar, Heading, VStack } from '@chakra-ui/react';
+import * as React from 'react';
+import { User } from '../../generated/graphql';
+
+interface Props {
+	user: Pick<User, "username">
+};
+
+const UserList: React.FC<Props> = ({ user }) => {
```

Comment:
i think spacing should be 2 to match other files

Reply:
whoops thought it was 2